packages/governance/src/types.js

```diff
@@ -1,77 +1,128 @@
 // @ts-check
 
 /**
- * @typedef { 'amount' | 'brand' | 'installation' | 'instance' | 'nat' | 'ratio' | 'string' | 'unknown' } ParamType
+ * @typedef { 'choose_n' | 'order' | 'weight' } ChoiceMethod
+ *  CHOOSE_N: This is "approval voting". The voter specifies some number of
+ *    positions, and is endorsing them equally.
+ *  ORDER: The voter assigns ordinal numbers to some of the positions. The
+ *    positions will be treated as an ordered list with no gaps.
+ *  WEIGHT: The voter assigns a distinct integer weight to some number of
+ *   positions. The weights are not required to be distinct or consecutive.
+ *   High numbers are most preferred.
  */
 
 /**
- * @typedef { Amount | Brand | Installation | Instance | bigint | Ratio | string | unknown } ParamValue
+ * @typedef { 'param_change' | 'election' | 'survey' } ElectionType
+ * param_change is very specific. Survey means multiple answers are possible,
+ * Election means some candidates are going to "win". It's not clear these are
+ * orthogonal. The important distinction is that param_change has a structured
+ * question, while the others have a question presented as a string.
  */
 
 /**
- * @typedef { 'choose_n' | 'order' | 'weight' } ChoiceMethod
+ * @typedef { 'half' | 'all' | 'none' } QuorumRule
```

Comment:
I haven't yet gotten to where they are used, so I'm not sure. But should `half` be `majority` and `all` be `unanimity`? `none` surprises me enough that I am probably leaping to the wrong meaning.